content/en/serverless/azure_app_services.md

```diff
@@ -65,20 +71,20 @@ The Datadog extension for Azure App Service provides additional monitoring capab
 
 ### Installation
 
-1. Configure the Azure integration to monitor your web app and verify it is configured correctly by ensuring that you see an `azure.app_service.count` metric tagged with the name of your web application. **Note**: This step is critical for metric/trace correlation, functional trace panel views in the Datadog portal, and accurate billing.
```

Comment:
Let's make "Azure Integration" a link to the azure integration docs (just like in the first step of the requirements section)